app/views/media_objects/_intercom_push_form.html.erb

```diff
@@ -17,17 +17,18 @@ Unless required by applicable law or agreed to in writing, software distributed
 <% @collections = session[:intercom_collections] || Avalon::Intercom.new(user_key).user_collections %>
 <% @collections.map! { |c| OpenStruct.new(c) } %>
 <div class="modal-header">
-  <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-  <h1><%= Settings.intercom['default']['push_label'] %></h1>
+   <h4 class='modal-title'><%= Settings.intercom['default']['push_label'] %></h4>
+   <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span
+      class="sr-only">Close</span></button>
 </div>
 
 <div class="modal-body">
   <div class="form-group">
     <label for "collection_id">Choose Target Collection:</label>
-    <div class="input-group">
+    <div class="input-group" style="flex-flow: nowrap">
       <%= select_tag :collection_id, options_from_collection_for_select(@collections, "id", "name", session[:intercom_default_collection] ), required: true, style: 'width:100%;', class:'form-control', form:'intercom_push_form' %>
       <span class="input-group-btn">
-        <button class="btn btn-secondary" title="Refresh list" onclick="getIntercomCollections(true, '<%= intercom_collections_media_objects_path %>');"><i class="fa fa-refresh"></i></button>
+        <button class="btn" style="lightgrey" title="Refresh list" onclick="getIntercomCollections(true, '<%= intercom_collections_media_objects_path %>');"><i class="fa fa-refresh"></i></button>
```

Comment:
I beleive this should be `class="btn btn-outline"` instead of `style="lightgrey"` which creates a white button with an outline.

Reply:
For reference this is what it looks like in dark avalon: 
![image](https://user-images.githubusercontent.com/1053603/169154398-6b4407e8-1407-4be6-a725-c4ce56768995.png)
And this is what it looks like with this PR:
![image](https://user-images.githubusercontent.com/1053603/169154690-bcfb5e99-9d89-471a-87ce-bfa6f2909bfd.png)
The `btn-secondary` didn't look good, but I think the white button works fine.